### DISCUSSION TITLE
Looking for help reading Roast Profiles on Artisan

### DISCUSSION THREAD

**Discussion by bagelsbrews** [artisan-roaster-scope/artisan#1111] [Q&A]
![IMG_4729](https://user-images.githubusercontent.com/122627463/220745193-887aba64-d530-44ad-8ca1-b72aca6014be.JPG)
![costa rica 3rd roast 12 lbs](https://user-images.githubusercontent.com/122627463/220745233-5f8c8098-0ad6-45c4-9d3a-f35c9f400d6d.png)

The curve on Artisan and the notebook recordings are the same roast.  There is an older probe that gives results that I wrote in notebook.  A newer probe is tied in with a phidget to the USB and shown on the computer display.

Notice the huge temperature difference.  Also, my graph doesn't look like what I' reading from others of what a good roast looks like.

Can you help me?

Thanks,

Michael Smith

**Comment by poundy** [comment]
There are multiple things going on here. Some, doesn't matter about, some certainly matter.

1) absolute temperature difference between two sets of probes in two different places in the roaster, using two different measurement systems. This doesn't matter - the probe is telling you "it's truth". If one probe says 300f, one says 320f, which one is right? There's a good chance they are both wrong! But they are their own version of correct. And if my probe is reading 320f, what does that mean to your roasting when yours reads 320f? Nothing..... 
You can either ignore any difference here, or if you are so inclined, you can adjust the probe through some formula. Figuring out the formula to make a new probe read roughly the same as some other probe is pretty hard work because you can't usually just add 15f to it to have it be "right", there's usually some non-linearity. But Artisan has symbolic adjustment capability to allow you to do that if you so choose.  
1a) there's also a chance you have selected the wrong probe type that causes an incorrect difference. Check your setup is correct based on what you bought.

2) your graph has one classic flaw - the temperature axis is compressing your data because you have left the lower point at 0, when your  TP is well over 250. Do yourself a massive favour, adjust that so that it shows you the data you collected, not a heap of whitespace.

3) you have a massive dip in the middle - what did you do? Whatever happened, it appears you set the roast back a fair way. That's one of the issues for this roast, I suspect, which may or may not get you acceptable coffee (only tasting will tell!)

4) none of us here, except you, know what you're comparing as a "good roast" graph. And TBH, a good roast is one that tastes good, not one whose graph looks <insert some descriptor here>. Someone telling you their graph is good doesn't make the coffee coming out the other side good, and vice versa. Taste is king.  But if you had some specifics, "my graph has this here that I never see on others", it might be possible to assist further...    
4a) it's pretty "obvious" that once in first crack, your bean temp ROR was increasing, and that is one obvious "rule" that many try to adhere to. That's likely just late or too small burner and airflow adjustments, that experience will tell you - and note that your manual recording does in fact show you a similar behaviour on the highlighted roast, it's just really (really!) obvious when you have a graph with much more data, and the ROR plotted, for you. Again, taste is king, not graphs and alignment with "rules", but most people aim to minimise that kind of increasing temperature. You need to find ways to manage that on your roaster.

**Comment by bagelsbrews** [comment]
1.   Probes are the same and both are selected accurately.
  2.  Fixed the graph.  Looks much better.  Thank you.
  3.  Not sure what I did here.  I think I probably went from cooling bin to all roasting drum air to slow down the roast.   Coffee ended up pleasant even making this mistake, but who knows how much better it would have been without the mistake?
  4.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
